src/EditorFeatures/VisualBasicTest/GenerateConstructor/GenerateConstructorTests.vb

```diff
@@ -1434,21 +1434,22 @@ End Enum
 Public Class MyAttribute
     Inherits System.Attribute
 
-    Private v1 As Short()
```

Comment:
This test change brings VB closer to the [equivalent C# test](http://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp.EditorFeatures.UnitTests/GenerateConstructor/GenerateConstructorTests.cs,b8dec7376531facc).